modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vmc/installing-restricted-network-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+ifeval::["{context}" == "installing-vmc"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-vmc-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-vmc-network-customizations"]
+:ipi:
+endif::[]
+
+[id="setting-up-vmc-for-vsphere_{context}"]
+= Setting up VMC for vSphere
+
+You can install {product-title} on VMware Cloud (VMC) on AWS hosted vSphere clusters to enable applications to be deployed and managed both on-premise and off-premise, across the hybrid cloud.
+
+image::150_OpenShift_VMware_on_AWS_0321_arch.svg["VMC on AWS Architecture"]
+
+You must configure several options in your VMC environment prior to installing {product-title} on VMware vSphere. Ensure your VMC environment has the following prerequisites:
+
+* Create a non-exclusive, DHCP-enabled, NSX-T network segment and subnet. Other virtual machines (VMs) can be hosted on the subnet, but at least eight IP addresses must be available for the {product-title} deployment.
+
+ifdef::ipi[]
+* Allocate two IP addresses, outside the DHCP range, and configure them with reverse DNS records.
+** A DNS record for `api.<cluster_name>.<base_domain>` pointing to the allocated IP address.
+** A DNS record for `*.apps.<cluster_name>.<base_domain>` pointing to the allocated IP address.
+endif::ipi[]
+
+* Configure the following firewall rules:
+** An ANY:ANY firewall rule between the {product-title} compute network and the Internet. This is used by nodes and applications to download container images.
```

Comment:
I still have one comment here:
For [Installing a cluster on VMC in a restricted network with user-provisioned infrastructure](https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-restricted-networks-vmc-user-infra.html), compute network should not have or less internet access, container images are pulling from local mirror registry server.

Reply:
@jinyunma I have added a conditional to skip the bullet on line 38 for restricted network installs.

Can you give a `lgtm` if this looks good? Thanks!